site_scons/site_tools/recinstall.py

```diff
@@ -58,21 +58,22 @@ def RecursiveInstall(env, target, dir):
 
     l = len(dir) + 1
 
-    relnodes = [ n.abspath[l:] for n in nodes ]
+    relnodes = [n.abspath[l:] for n in nodes]
```

Comment:
Should this be changed to `[1: ]` to be consistent with the other formatting  changes?